Plans: Fix link to more info about plugins

nylen opened the pull request:
The Plans page text says "All plans already come with a custom set of plugins tailored just for them. Check out all included plugins" and then links to `/plugins` within Calypso, which is only useful for Business users.

Let's fix this - there's a support page that's perfect to link to instead.  https://en.support.wordpress.com/plugins/ says:

> On WordPress.com, we include the most popular plugin functionality within our sites automatically.
>
> Additionally, the Business plan allows you to choose from many thousands of plugins, and install them on your site.

| Before | After |
|--|--|
| ![2018-05-02t15 03 07-0500](https://user-images.githubusercontent.com/227022/39546474-2622f59c-4e1a-11e8-9623-4641220c7dbb.png) | ![2018-05-02t15 01 51-0500](https://user-images.githubusercontent.com/227022/39546479-2d5597ac-4e1a-11e8-867e-6cf0beb0b79e.png) |

### Testing instructions

Visit `/help` and verify that things still work, in particular the localized links to the support site like "Get Started".

Visit `/plans/:site` and verify that the "Find out more about plugins" link (near the bottom) now points to a support page about plugins and opens in a new tab.

Change your user locale to Spanish (under `/me/account`) and note that this link appears in Spanish, with support content in Spanish.

Change locale to Brazilian Portuguese and note that this link points to https://pt-br.support.wordpress.com/plugins/ which redirects to https://br.support.wordpress.com/plugins/.  Content appears in English until this page can be updated.

kwight commented on client/my-sites/plans-features-main/wpcom-faq.jsx:
```diff
@@ -52,9 +52,9 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
+						' {{a}}Find out more about plugins{{/a}}.',
 					{
-						components: { a: <a href={ `/plugins/${ siteSlug }` } /> },
+						components: { a: <a href="https://en.support.wordpress.com/plugins/" /> },
```
I don't think we want to hardcode the `en` English version here; we should be checking the locale of the current user and plugging that in (eg. es.support.wordpress.com`). If the site doesn't exist, eg. `fr.support.wordpress.com`, it'll forward to `en` anyways. Some recent background discussion can be found at pxLjZ-4Cu-p2.

nylen replied:
Good point.

I did a bit more digging and found a few things.

1. It looks like this is in a pretty bad state across Calypso.  The majority of [instances of `support.wordpress.com`](https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=support.wordpress.com&type=) link to the `en` subdomain.
2. There are some links to plain `support.wordpress.com` in there, but I would like to know more about what this does before using it.
3. There is a `getSupportLocale` method which could be extracted out to a utility library and re-used.  It will return one of (`en`, `es`, `pt-br`) which I gather are the locales we think are the best-translated.  BUT:
4. https://es.support.wordpress.com/es-plugins/ has no mention of the Business plan allowing plugin support.  This makes sense given that we don't have a strategy for keeping support page content up to date.
5. The content of https://pt-br.support.wordpress.com/plugins/ is in English anyway.

So overall, I think it might actually best to link to the English version here.  For non-English users, browsers should offer to translate it automatically, which is going to be better than what we can offer at this point.

Another option would be to update the Spanish translation for this change and go with `getSupportLocale` - I would be fine with that too.

nylen commented:
Updated (see [https://github.com/Automattic/wp-calypso/pull/24636#pullrequestreview-117154660](review comments above)):

- Pull out existing `getSupportLocale` function into a reusable library.
- Translate the new link to the plugins support page where possible (effectively this only works for `es`, and might work for `pt-br` if this content is translated one day).
- Open the support link in a new tab, like what's done in other parts of Calypso.
- Added testing instructions in the original post.

jonathansadowski commented on client/lib/i18n-utils/utils.js:
```diff
@@ -136,3 +137,20 @@ export function removeLocaleFromPath( path ) {
+ *
+ * @returns {string} A slug which is a valid subdomain of *.support.wordpress.com.
+ */
+export function getSupportLocale() {
```
Nice change! This should be helpful for linking to the correct documentation!

nylen commented:
Thanks for reviewing.  I agree, let's ship this and refine later if needed.

lancewillett commented:
Thank you for the quick fixes on this — especially @nylen. 🌟
